projects/arlas-toolkit/src/lib/components/config-manager/action-modal/action-modal.component.ts

```diff
@@ -100,6 +100,17 @@ export class ActionModalComponent {
       );
   }
 
+  public create(name: string) {
```

Comment:
What is this used for ?

Reply:
it's present in the template; i removed by error
and the compilation didn't warn about it missing